src/coreclr/src/System.Private.CoreLib/src/System/Reflection/RuntimeParameterInfo.cs

```diff
@@ -481,12 +483,16 @@ private static DateTime GetRawDateTimeConstant(CustomAttributeData attr)
 
         public override Type[] GetRequiredCustomModifiers()
         {
-            return m_signature.GetCustomModifiers(PositionImpl + 1, true);
+            return m_signature is null ?
```

Comment:
Is this fixing a bug? Are we missing a test?

Reply:
I believe so. Just need to construct one.

Reply:
Updated an existing test such that it would null ref before the change and pass after.